### PR TITLE
Switch to all HTTPS URIs

### PR DESCRIPTION
**Pull request by mebersol**

### CODE REVIEW

File: windows-server-container-tools/Install-ContainerHost/Install-ContainerHost.ps1 [hunk 1]
```diff
@@ -77,7 +77,7 @@ param(
 
     [string]
     [ValidateNotNullOrEmpty()]
-    $WimPath = "http://aka.ms/ContainerOSImage"
+    $WimPath = "https://aka.ms/ContainerOSImage"
```
Comment (StefanScherer) [inline windows-server-container-tools/Install-ContainerHost/Install-ContainerHost.ps1]: This URL is redirected back to an http URL http://download.microsoft.com/download/...

Comment (mebersol) [inline windows-server-container-tools/Install-ContainerHost/Install-ContainerHost.ps1]: Apparently DLC doesn't have https links...

File: windows-server-container-tools/New-ContainerHost/New-ContainerHost.ps1 [hunk 2]
```diff
@@ -89,7 +89,7 @@ param(
     [Parameter(ParameterSetName="Staging", Mandatory=$true)]
     [string]
     [ValidateNotNullOrEmpty()]
-    $VhdPath = "http://aka.ms/containerhostvhd",
+    $VhdPath = "https://aka.ms/containerhostvhd",
```
Comment (StefanScherer) [inline windows-server-container-tools/New-ContainerHost/New-ContainerHost.ps1]: This URL is redirected back to an http URL http://download.microsoft.com/download/...

File: windows-server-container-tools/New-ContainerHost/New-ContainerHost.ps1 [hunk 3]
```diff
@@ -242,7 +242,7 @@ Cache-HostFiles
 
     if ($global:DeveloperMode -and ($global:localVhdName -match "\.th2_release\."))
     {
-        Copy-File -SourcePath "http://aka.ms/containerzdp" -DestinationPath "$global:localVhdRoot\zdp.cab"
+        Copy-File -SourcePath "https://aka.ms/containerzdp" -DestinationPath "$global:localVhdRoot\zdp.cab"
```
Comment (StefanScherer) [inline windows-server-container-tools/New-ContainerHost/New-ContainerHost.ps1]: This URL is redirected back to an http URL http://10.137.119.43/Windows10.0-KB3081439-x64.cab

**Pull request merged**